packages/lib/contracts/upgradeability/Proxy.sol

```diff
@@ -17,6 +17,10 @@ abstract contract Proxy {
     _fallback();
   }
 
+  receive () payable external {
```

Comment:
why having both `receive` and `fallback` functions..as fallback can receive ether as well ? https://solidity.readthedocs.io/en/latest/contracts.html#fallback-function 
@spalladino @ben-kaufman

Reply:
Solidity gives a warning if you don't have the receive too, and for a proxy contract, you must have the fallback.